e2e/report-user.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+//TODO: run email server?
```

Comment:
```suggestion
```
It's not strictly necessary, since the test will pass either way.

It might be worth requiring it and checking that `flash.report-sent` appears at the end of the test.